src/ros/update.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd ~/ros2_ws/
+sudo rm -R ~/ros2_ws/install/ ~/ros2_ws/build/ ~/ros2_ws/log/
```

Comment:
could this cause issues if user isn't root?

Reply:
I'm in the user where it's not root yet. I think it should be fine. 

It would just ask us to type the password for it to run. Do you think this would still cause some issue?